app/assets/stylesheets/layouts/_results.scss

```diff
@@ -19,6 +19,33 @@
   }
 }
 
+.l-results__firm-name {
+  @include column(12);
+
+  margin-left: 0;
```

Comment:
Put this PR to WIP just now. I thought I'd taken all of these out and replaced with `--margin-collapse` modifiers. Will review tomorrow and tidy up. Perhaps the `row` mixin is what's required as you say.